js/index.js

```diff
@@ -1,6 +1,11 @@
+import data from './projectData.js';
+
 const menuBtn = document.getElementById('menu-btn');
 const closeBtn = document.getElementById('close-btn');
 const menu = document.getElementById('menu');
+const projectContainer = document.getElementById('projects-container');
+
+const popup = document.getElementById('popup');
```

Comment:
- [ ] Kindly make the close button visible to the mobile version of the pop-up card.

Your design
![image](https://user-images.githubusercontent.com/81932478/189950455-8af788c0-55ab-4ee1-8828-b7f06958d4f8.png)

Figma design
![image](https://user-images.githubusercontent.com/81932478/189950565-8dd73616-c745-4255-9e63-74ef031046d3.png)

- [ ] Kindly align the buttons in the mobile version to match the given design
- [ ] Kindly style the languages to match the given design

Your design
![image](https://user-images.githubusercontent.com/81932478/189951218-1d09d324-9f20-406f-9d43-6d280b45b632.png)

Figma design
![image](https://user-images.githubusercontent.com/81932478/189951324-303d73a7-1b77-41c6-a0ec-b7c04592ef50.png)

Reply:
Sorry, but If I make it visible by changing the style it will not correspond to the template design!

Reply:
![image](https://user-images.githubusercontent.com/11340240/190083460-e5193fa0-1499-4b49-81a7-7e102f049b22.png)
It will be okay like this ?